File: Patient.py
import cgi
class Patient:
    def __init__(self, fname, lname, patientID, status):
        self.fname = fname
        self.lname = lname
        self.patientID = patientID
        self.status = status
       
    
    def __str__(self):
        return f"First name: {self.fname}\tLast Name: {self.lname}\tID: {self.patientID}\t Status: {self.status}"

    def getPatientID(self):
        return self.patientID
    
    def getFname(self):
        return self.fname

    def getPatientLname(self):
        return self.lname

    def getPatientStatus(self):
        return self.status

File: js/Patient.js
class Patient {
    constructor (fname, lname, id, status) {
        this.fname = fname
        this.lname = lname
        this.id = id
        this.status = status
    }

    get fname () {
        return this.fname
    }
    get lname () {
        return this.lname
    }
    get id () {
        return this.id
    }
    get status () {
        return this.status
    }

    patient_list = []

    addPatient (patient){
        patient_list.push(patient)
    }

    createPatient (fname, lname, id, status) {
        let patient = Patient(fname, lname, id, status)
        addPatient(patient)
    }
    

}

